content-src/asrouter/templates/Trailhead/_Trailhead.scss

```diff
@@ -267,6 +267,10 @@
   text-align: center;
   transition: max-height 0.5s $photon-easing;
 
+  // This is needed for the transition to work, but will cut off content at the smallest breakpoint
+  @media (min-width: $break-point-medium) {
+    max-height: 1000px;
```

Comment:
Ah. @AllegroFox removed this in #5158, so I guess no transition for narrow but this is quite the corner case? We could just increase the max-height to say 1100px with no media query ?

Reply:
That's right. I think the transition would probably look really weird at the smallest screen anyway so I think the media query makes sense